omnigibson/scenes/scene_base.py

```diff
@@ -24,17 +34,28 @@
 )
 from omnigibson.utils.registry_utils import SerializableRegistry
 from omnigibson.utils.ui_utils import create_module_logger
-from omnigibson.utils.usd_utils import CollisionAPI
+from omnigibson.utils.usd_utils import CollisionAPI, add_asset_to_stage
 
 # Create module logger
 log = create_module_logger(module_name=__name__)
 
 # Create settings for this module
 m = create_module_macros(module_path=__file__)
 
+# Margin between the AABBs of two different scenes.
+m.SCENE_MARGIN = 10.0
+m.INITIAL_SCENE_PRIM_Z_OFFSET = -100.0
```

Comment:
what is this? where the initial scene prim is placed wrt the world?

Reply:
when we load a new scene with existing ones, since we don't know its size yet, we wouldn't know where to position it to avoid colliding with the previous ones. This offset here puts it at -100 on the z axis so that we can wait until it's loaded here and reposition it with it aabb.